src/content/developers/docs/gas/index.md

```diff
@@ -36,11 +36,37 @@ Although a transaction includes a limit, any gas not used in a transaction is re
 ![Diagram showing how unused gas is refunded](../transactions/gas-tx.png)
 _Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
 
+## Why are gas fees so high? {#Why-are-gas-fees-so-high}
+
+High Gas fees are due to the popularity of Ethereum. As every operation on the Etheruem blockcahin uses gas, including computation, checking the balance of an account and transaction fees, these all contribute to the high gas levels we experience. Gas cost alone does not actually determine how much we have to pay for a particular transaction. To calculate the transaction fee we have to multiply the gas cost by gas price. (Which is measured in GWEI)
```

Comment:
Few typos in here... "High **g**as fees ..." (lowercase gas), "blockc**ha**in"
Not sure the second sentence is quite accurate... "checking the balance of an account" - this typically doesn't gas just to call a balance. And, "and transaction fees" - Not sure I understand you correctly here... gas _is_ the transaction fee, it isn't an operation by itself.

My thoughts on this would be more two fold- explain that "gas" units are roughly a unit of computation complexity, and then explain that since gas (aka computation) in a block is capped, then as demand goes up, price per gas unit (paid in small amounts of ether, gwei)

### Some suggestions:
To perform any operation on Ethereum requires consuming gas. This includes calculations, storing or manipulating data, or transferring tokens, each consuming different amounts of "gas" units. As dapp functionality grows more complex, the number of operations a smart contract performs grows too, and all contribute to high amounts of gas that must be paid for.

Gas is paid for with small amounts of ether (denominated in gwei, or 10<sup>-9</sup> ETH). This value represents the amount of ether being paid per "unit of gas", not for the entire transaction. Since gas is capped within a block, under the current system, this means that users must out-bid one another to be included in the next block if demand exceeds capacity.

Reply:
I have corrected the typo's and added in the information you gave, appreciate it!.

### My changes:

High gas fees are due to the popularity of Ethereum. To perform any operation on Ethereum requires consuming gas. This includes calculations, storing or manipulating data, or transferring tokens, each consuming different amounts of "gas" units. As dapp functionality grows more complex, the number of operations a smart contract performs grows too, and all contribute to high amounts of gas that must be paid for.

Gas is paid for with small amounts of ether (denominated in gwei, or (10<sup>-9</sup> ETH)). This value represents the amount of ether being paid per "unit of gas", not for the entire transaction. Since gas is capped within a block, under the current system, this means that users must out-bid one another to be included in the next block if demand exceeds capacity. Gas cost alone does not actually determine how much we have to pay for a particular transaction. To calculate the transaction fee we have to multiply the gas cost by gas price. (Which is measured in gwei)